styles.css

```diff
@@ -10,3 +10,6 @@
   color: green;
   font-weight: bold;
 }
+#heading2{
```

Comment:
Can we add an empty line between the selector?

Reply:
Done